src/Lib.P2P.Tests/Cryptography/CtrStreamCipherTest.cs

```diff
@@ -0,0 +1,101 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System.Text;
+using Lib.P2P.Cryptography;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using MultiFormats;
+using Org.BouncyCastle.Crypto.Engines;
+using Org.BouncyCastle.Crypto.Parameters;
+
+namespace Lib.P2P.Tests.Cryptography
+{
+    [TestClass]
+    public class CtrStreamCipherTest
+    {
+        [TestMethod]
+        public void SameAsNodeJS()
+        {
+            var key = new byte[32];
+            var iv = new byte[16];
+            var encrypt = new CtrStreamCipher(new AesEngine());
+            var p = new ParametersWithIV(new KeyParameter(key), iv);
+            encrypt.Init(true, p);
+
+            var plain = new byte[] {(byte) 'a'};
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)